### PR TITLE
Changelog for 2.5-beta

### PR DESCRIPTION
**Pull request by daschuer**
More changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -220,6 +229,7 @@
 * Fix ambiguous overload error due to native qDebug impl for std::optional
   [#12981](https://github.com/mixxxdj/mixxx/issues/12981)
 * Fix 'selected click' bug [#12488](https://github.com/mixxxdj/mixxx/pull/12488)
```
Comment (ronso0) [inline CHANGELOG.md:231]:
```suggestion
* Workaround for Qt6 'selected click' bug [#12488](https://github.com/mixxxdj/mixxx/pull/12488)
```

**Comment by daschuer** [comment]
Fixed. I have added also the latest merge.

File: CHANGELOG.md [hunk 2]
```diff
@@ -77,6 +82,9 @@
 * Add Compressor effect [#12523](https://github.com/mixxxdj/mixxx/pull/12523)
 * add Glitch effect [#11329](https://github.com/mixxxdj/mixxx/pull/11329)
 * Add backend for Audio Unit (AU) plugins on macOS [#12112](https://github.com/mixxxdj/mixxx/pull/12112)
+* Effect Meata Knob: Draw arc from default meta position
```
Comment (ronso0) [inline CHANGELOG.md:85]:
```suggestion
* Effect Meta knob: Draw arc from default meta position
```

Comment (ronso0) [reply CHANGELOG.md:85]: Just this nit, feel free to squash and merge yourself.

File: CHANGELOG.md [hunk 3]
```diff
@@ -298,6 +313,7 @@
 * Code Style: Add branches around single line blocks. [#13097](https://github.com/mixxxdj/mixxx/pull/13097)
 * Add missing member in copy ctor [#13229](https://github.com/mixxxdj/mixxx/pull/13229)
 * Refactor/preferences enums [#12798](https://github.com/mixxxdj/mixxx/pull/12798)
+* (opti/cleanup) make use of WLibrary::getCurrentTrackTableView() [#13335](https://github.com/mixxxdj/mixxx/pull/13335)
```
Comment (ronso0) [inline CHANGELOG.md:316]: I used that prefix only as tag, let's remove it.
And move that PR to the Library section?
```suggestion
* Library control: make use of WLibrary::getCurrentTrackTableView() [#13335](https://github.com/mixxxdj/mixxx/pull/13335)
```

**Comment by daschuer** [comment]
I will merge this now, to not have to deal with conflicts after merging 2.4 to 2.5 and main. The CHANGLOG is anyway a subject of continuous changes.

**Pull request merged**